src/flake8_json_reporter/reporters.py

```diff
@@ -83,6 +84,9 @@ def stop(self):
         if self.files_reported_count > 0:
             self.write_line("\n")
         self.write_line("}\n")
+        super(
+            DefaultJSON, self
+        ).stop()  # DefaultJSON.stop would write and extra close brace
```

Comment:
the formatter hated the inline comment here -- maybe move it above?  this also seems a little fragile ("grandparent" super) -- perhaps it should explicitly call `BaseFormatter.stop(self)` since it's more obvious what this is doing? (rather than looking like a mistake)